### ISSUE TITLE
User isn't able to add white spaces in their names

### ISSUE THREAD

**Issue by abhi-status** [status-im/status-desktop#5364]
# Bug Report

## Description

Currently, the user cannot add white spaces to their display names in the onboarding screen.

From John -
>  If allowing spaces in user names doesn't cause any issues, then we should allow spaces.  But we should check the @ mention auto complete behaviour if we make this change.
> e.g. spaces in names is 'nice to have', but not essential

#### Expected behavior
User must be allowed to continue with onboarding having white spaces in their names

#### Actual behavior
User is blocked by the validation
![image](https://user-images.githubusercontent.com/95302135/161762615-b79b6fd4-b102-4031-b519-1b3de026390d.png)

### Additional Information

- Status desktop version: Master : b54111f82ac0bdbecdace09358080f53891a5505

**Comment by iurimatias** [comment]
https://github.com/status-im/feature-specs/pull/42/files#r809295391

**Comment by jrainville** [comment]
There https://discord.com/channels/624205794384281629/730410574428504166/943970998942859315
> re. the usernames question in https://github.com/status-im/feature-specs/pull/42 , I think we should probably disallow spaces in usernames because allowing spaces would make our lives harder when it comes to things like autocomplete.

**Comment by John-44** [comment]
@iurimatias @jrainville If I remember correctly we don't have spaces because of if issues with the @ mention auto-complete. 



> I think that this is normal. We decided to disable white spaces to limit problems and also limit the chance that someone imitates a 3 word name. cc @John-44

Issues with the @ mention auto-complete was the issue.  If these can be fixed, or if they aren't a problem any more it would be great to have spaces in usernames. 

There are other protections we can use against someone imitating a 3 word name, and in any case we don't have that many users today and in the future it won't be possible to create a 3 word name so I'm not worried about this (e.g. we shouldn't block on letting users have spaces in their usernames because of worries about imitating a 3 word name)
